brion/plugin/compartmentReportBinary.cpp

```diff
@@ -662,29 +689,31 @@ bool CompartmentReportBinary::_parseHeader()
     if (_tunit.empty())
         _tunit = "ms";
 
+    // After parsing the header we figure out the end of the mapping by reading
+    // the data offset of the first cell.
+    _dataOffset = get<uint64_t>(ptr, DATA_INFO + _header.headerSize);
```

Comment:
why did this move out of _parseMapping()? It's always the same for multiple updateMapping() calls

Reply:
(I thought I had already answered this, but I can't see the comment)
Yes, in fact it should the data offset should be computable just with the information from the header, but it doesn't happen to be the case (or I didn't spend enough time figuring it out).

Reply:
still no satisfying answer why you had to move the code. Where it was before should still work, no?

Reply:
It can't be at the original place. The offset is needed for remapping the file (line 282) and then read the GIDs, which is done even if the full mapping is not processed.